src/views/blocks/tickets.php

```diff
@@ -62,67 +63,60 @@
 	add_filter( 'tribe_tickets_order_link_template_already_rendered', '__return_true' );
 }
 ?>
-<form
-	id="tribe-tickets"
-	action="<?php echo esc_url( $provider->get_cart_url() ); ?>"
-	<?php tribe_classes( $cart_classes ); ?>
-	method="post"
-	enctype='multipart/form-data'
-	data-provider="<?php echo esc_attr( $provider->class_name ); ?>"
-	autocomplete="off"
-	data-provider-id="<?php echo esc_attr( $provider->orm_provider ); ?>"
-	novalidate
->
-	<h2 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__title"><?php echo esc_html( tribe_get_ticket_label_plural( 'event-tickets' ) ); ?></h2>
-	<input type="hidden" name="tribe_tickets_saving_attendees" value="1"/>
-	<input type="hidden" name="tribe_tickets_ar" value="1"/>
-	<input type="hidden" name="tribe_tickets_ar_data" value="" id="tribe_tickets_block_ar_data"/>
-	<?php $this->template(
-		'components/notice',
-		[
-			'id' => 'tribe-tickets__notice__tickets-in-cart',
-			'notice_classes' => [
-				'tribe-tickets__notice--barred',
-				'tribe-tickets__notice--barred-left',
-			],
+<div <?php tribe_classes( $classes ); ?>>
+	<form
+		id="tribe-tickets"
+		action="<?php echo esc_url( $provider->get_cart_url() ); ?>"
+		class="tribe-tickets"
+		class="tribe-tickets"
+		method="post"
+		enctype='multipart/form-data'
+		data-provider="<?php echo esc_attr( $provider->class_name ); ?>"
+		autocomplete="off"
+		data-provider-id="<?php echo esc_attr( $provider->orm_provider ); ?>"
+		novalidate
+	>
+		<h2 class="tribe-common-h4 tribe-common-h--alt tribe-tickets__title">
+			<?php echo esc_html( tribe_get_ticket_label_plural( 'event-tickets' ) ); ?>
+		</h2>
+		<input type="hidden" name="tribe_tickets_saving_attendees" value="1"/>
+		<input type="hidden" name="tribe_tickets_ar" value="1"/>
+		<input type="hidden" name="tribe_tickets_ar_data" value="" id="tribe_tickets_block_ar_data"/>
+		<?php $this->template(
+			'components/notice',
+			[
+				'id' => 'tribe-tickets__notice__tickets-in-cart',
+				'notice_classes' => [
+					'tribe-tickets__notice--barred',
+					'tribe-tickets__notice--barred-left',
+				],
+
+				'content_classes' => [
+					'tribe-common-b3',
+				],
+				'content' => __( 'The numbers below include tickets for this event already in your cart. Clicking "Get Tickets" will allow you to edit any existing attendee information as well as change ticket quantities.', 'event-tickets' )
```

Comment:
* :warning: Array double arrow not aligned correctly; expected 9 space(s) between "'content'" and double arrow, but found 1.
* :no_entry_sign: Each line in an array declaration must end in a comma
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: a5a0b7f277fc35855ccc455a901c8200</sub>